### PR TITLE
Design review of Clubs page.

### PR DESCRIPTION
**Pull request by vazquez**

### CODE REVIEW

**Comment by toolness** [comment]
@vazquez, did #277 create a merge conflict w/ this? It says I can't automatically merge the PR...

**Comment by vazquez** [comment]
Yes, will fix it right now :)

**Comment by vazquez** [comment]
@toolness Done!

File: less/components/index.less [hunk 1]
```diff
@@ -8,3 +8,4 @@
 @import "page";
 @import "page-end-cta";
 @import "home-page";
+@import "clubs-page";
```
Comment (toolness) [inline less/components/index.less]: Huh, what happened to this? Did it go away in an earlier commit but we forgot to remove the `@import` until now?

**Comment by toolness** [comment]
If you can answer my above [question](https://github.com/mozilla/teach.webmaker.org/pull/275#discussion_r26400845) and travis likes this, it's good to merge!

File: less/components/index.less (hunk 1 above)
Comment (vazquez) [inline less/components/index.less]: No, I added a new partial called clubs-page.less to style Clubs specific CSS.

**Comment by toolness** [comment]
Ah cool!

**Pull request merged**